.jenkins/pytorch/win-test.sh

```diff
@@ -57,9 +57,9 @@ fi
 
 run_tests() {
     # Run nvidia-smi if available
-    for path in  /c/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi.exe /c/Windows/System32/nvidia-smi.exe; do
-        if [ -x $path ]; then
-            $path;
+    for path in  "/c/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi.exe" "/c/Windows/System32/nvidia-smi.exe"; do
```

Comment:
Thanks for fixing this one. Maybe you should do it in a separate PR.

Reply:
will do